### DISCUSSION TITLE
Issue with GitHub Copilot Access - Request for Assistance.

### DISCUSSION THREAD

**Discussion by myamin66** [community/community#43644] [Copilot Conversations]
### Select Topic Area

General

### Body

Dear GitHub Copilot Support Team,

I am writing to inquire about an issue I encountered with my GitHub copilot account. I am a verified student and currently have a GitHub Pro account. However, I recently received a message stating that my extended access to the GitHub Copilot Technical Preview has ended.

I attempted to resolve the issue by logging out and logging back in, but the issue persists. I would greatly appreciate any assistance in resolving this issue so that I can continue to use my GitHub copilot account as normal.

Thank you in advance for your help.

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Reply by myamin66** [reply]
@otkrickey 
I am writing to inquire about an issue I encountered with my GitHub copilot account. I am a verified student and currently have a GitHub Pro account. However, I recently received a message stating that my extended access to the GitHub Copilot Technical Preview has ended.

I attempted to resolve the issue by logging out and logging back in, but the issue persists. I would greatly appreciate any assistance in resolving this issue so that I can continue to use my GitHub copilot account as normal.

**Reply by otkrickey** [reply]
We are sharing various examples of this issue that occurred today.
Please check the link tag below.
I hope I can be of help to you.
#43634

**Comment by otkrickey** [comment]
Solved: #43634
